server/job/plexsync/index.ts

```diff
@@ -89,6 +89,12 @@ class JobPlexSync {
             newMedia.tmdbId = Number(tmdbMatch);
           }
         });
+        if (newMedia.imdbId && !newMedia.tmdbId) {
+          const tmdbMovie = await this.tmdb.getMovieByImdbId({
+            imdbId: newMedia.imdbId,
+          });
+          newMedia.tmdbId = tmdbMovie.id;
+        }
```

Comment:
What about one more check following this to make sure we actually got the tmdb id? Just in case we couldn't find a match for some reason?

Reply:
Added code to throw error in case tmdb id is still unknown after these checks.